libraries/abstractions/mqtt_agent/mqtt_agent_interface.cmake

```diff
@@ -0,0 +1,36 @@
+# MQTT Agent Interface
+afr_module(NAME mqtt_agent_interface INTERNAL)
+
+set(src_dir "${CMAKE_CURRENT_LIST_DIR}")
+
+set(include_dir "${CMAKE_CURRENT_LIST_DIR}/include")
+
+# Include filepaths for source and include.
+include( ${CMAKE_CURRENT_LIST_DIR}/transport_interface.cmake )
```

Comment:
Inclusion of `transport_interface.cmake` shouldn't be required

Reply:
Removed